quickstart/src/index.js

```diff
@@ -7,32 +7,60 @@ var previewTracks;
 var identity;
 var roomName;
 
-// Attach the Tracks to the DOM.
+
+// Attach the Track to the DOM.
+function attachTrack(track, container) {
+  container.appendChild(track.attach());
+}
+
+// Attach array of Tracks to the DOM.
 function attachTracks(tracks, container) {
-  tracks.forEach(function(track) {
-    container.appendChild(track.attach());
+  tracks.forEach(function (track) {
+    attachTrack(track, container);
   });
 }
 
-// Attach the Participant's Tracks to the DOM.
-function attachParticipantTracks(participant, container) {
-  var tracks = getTracks(participant);
-  attachTracks(tracks, container);
+// Detach given track from the DOM
+function detachTrack(track) {
+  track.detach().forEach(function (element) {
+    element.remove();
+  });
 }
 
-// Detach the Tracks from the DOM.
-function detachTracks(tracks) {
-  tracks.forEach(function(track) {
-    track.detach().forEach(function(detachedElement) {
-      detachedElement.remove();
-    });
+// A new RemoteTrack was published to the Room.
+function trackPublished(publication, container) {
+  if (publication.isSubscribed) {
+    attachTrack(publication.track, container);
+  }
+  publication.on('subscribed', function (track) {
+    attachTrack(track, container);
   });
+  publication.on('unsubscribed', detachTrack);
+}
+
+// A RemoteTrack was unpublished from the Room.
+function trackUnpublished(publication) {
```

Comment:
@makarandp0 I think here you can just log the event. The track will already be detached since "unsubscribed" is fired first and then "trackUnpublished" is fired.